src/print.rs

```diff
@@ -266,18 +266,18 @@ impl BacktraceFrameFmt<'_, '_, '_> {
         // Filename/line are printed on lines under the symbol name, so print
         // some appropriate whitespace to sort of right-align ourselves.
         if let PrintFmt::Full = self.fmt.format {
-            write!(self.fmt.fmt, "{:1$}", "", HEX_WIDTH)?;
+            write!(self.fmt.fmt, "{:HEX_WIDTH$}", "")?;
         }
         write!(self.fmt.fmt, "             at ")?;
```

Comment:
If this part is going to be diffed, surely there's a better to write it, in general? The part you diffed is literally formatting the empty string in, and all this is to generate what is mostly whitespace...